student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [Optional] There's no reason to assign `parent_permission: parent_permission` since this is already done in the super class `Person`. We just need to reuse the field `parent_permission` as you have done with `age `and `name`.